tensorflow/compiler/xla/service/gpu/tests/gpu_ldg_test.cc

```diff
@@ -38,6 +38,11 @@ class GpuLdgTest : public GpuCodegenTest {};
 
 // Parameters are never overwritten, so parameter reads should get ld.global.nc
 // reads.
+//
+// On the ROCM platform the "ptx" string is not populated for the compiled
```

Comment:
Should we just disable these tests for ROCM?

Reply:
We still wanted to keep the "Compile" part of the "CompileAndVerifyPtx" and hence the comment, instead of disabling it altogether

something is better than nothing :)